compiler/daml-extension/package.json

```diff
@@ -152,5 +152,8 @@
         "@types/vscode": "1.39",
         "typescript": "3.6.3",
         "vsce": "1.88.0"
+    },
+    "resolutions": {
```

Comment:
Please only use resolutions when we have to force a version because a dependency has garbage bounds. in this case this is pulled in via a bound `    css-what "^5.0.0"` so we don’t have to mess with any bounds. Just deleting the yarn lock file and regenerating should do the trick.